src/Veldrid/D3D11/D3D11Texture.cs

```diff
@@ -143,6 +143,12 @@ public D3D11Texture(Device device, ref TextureDescription description)
             }
         }
 
+        public D3D11Texture(Texture2D existingTexture, TextureType type):
```

Comment:
Can you explain what some of these additional Texture-related changes are for?

Reply:
In certain citation I have to create a texture provided from external source. I've used this to create a texture with correct format.

I can actually roll back this as o have to move the format resolution code out of main Veldrid repository.

Reply:
@mellinoe fixed